src/GO/Job.php

```diff
@@ -361,7 +375,8 @@ public function run()
         if (is_callable($compiled)) {
             $this->output = $this->exec($compiled);
         } else {
-            $this->output = exec($compiled);
+            $outputArray = null;
+            $this->output = exec($compiled, $outputArray, $this->returnCode);
```

Comment:
Will this force every job to run in foreground?
It seems like a few tests are now failing, the tests that ensure that jobs can run in background.